networklibrary/src/main/java/com/eis/smsnetwork/SMSNetworkManager.java

```diff
@@ -38,41 +54,73 @@
         return netDictionary;
     }
 
+    /**
+     * @return A {@link NetSubscriberList} containing Peers who were invited to join the network and
+     * haven't answered yet.
+     */
+    public NetSubscriberList<SMSPeer> getInvitedPeers() {
+        return invitedPeers;
+    }
+
     /**
      * Starts a setResource request to the net
      *
-     * @param key                 The key identifier for the resource.
-     * @param value               The identified value of the resource.
+     * @param key                 The key identifier for the resource. It cannot have a backslash as
+     *                            its last character.
```

Comment:
> It cannot have a backslash as its last character.

That is pretty specific lol

Reply:
Oh damn it, I forgot to remove this.
You'll see in a later pull request, this is needed because we (and by we I mean @giovannivelludo ) changed what can and cannot be a key and a resource.
It's kinda complex and out of this pr, so don't look at this *yet*

Reply:
Before that we forbid keys and resources from containing a certain character (spaces) at all, so this is less restrictive. This was a side effect of some changes I made to enable sending multiple keys and resources in one SMS.